mrcnn/README.md

```diff
@@ -0,0 +1,98 @@
+# Model Training
+
+## Table of Contents
+
+1. [Introduction](#introduction)  
+1. [Examples](#examples)  
+1. [Usage](#usage)  
+ 
+
+## Introduction
+
+
+## Setup
+
+Run data syntheszer module to generate the training  datasets in the following folder structure:
+
+~~~~~~~
+        project
+          |-- mrcnn
+               |-- scipts
+                    |-- config.py
+                    |-- model.py
+                    |-- train.py
+                    |-- pre_process.py
+                    |-- requirements.txt
+                    |-- utils.py
+                    |-- visualize.py
+                    |-- Inference_notebook.ipynb
+                    |-- utils.py
+                    
+               |--- mask_data
+                    |-- id_map.json
+                    |-- logs/
+                    |-- mask_rcnn_coco.h5
+                    |-- test_image
+                    |-- train_image
+~~~~~~~
+
+## Training
+
+
+### Pre-processing
+Data pre-processing using pre_process.py to generate .h5 file for masks.
+
+
+### Model and Training
+
+```
+ -- Modified Matterport's implementation of Mask-RCNN deep neural network for object instance segmentation.
```

Comment:
If you replace all `--` with `*`, these will appear as proper bullet points

Reply:
Thanks for this, updated the file now with bullets.